packages/now-cli/src/util/deploy/create-deploy.js

```diff
@@ -38,6 +38,11 @@ export default async function createDeploy(
       throw new ERRORS_TS.DomainVerificationFailed(error.value);
     }
 
+    // If the domain isn't owned by the user
+    if (error.code === 'not_domain_owner') {
```

Comment:
This error code comes from the server-side? I wonder if we should update it to a more consistent "subject_reason" formatting (i.e. `deployment_not_found`). So `domain_not_owner` in this case?

Reply:
Also, it would be great if we could somehow share types between server and client so that we can perform an exhaustive check of all the possible error codes.